test/app/add_folder_words_page/add_folder_words_page_test.dart

```diff
@@ -43,8 +45,8 @@ void main() {
       whenListen(
         mockAddFolderWordsCubit,
         Stream.fromIterable(
-          const [
-            AddFolderWordsState.loaded([mockedWordEntityV1]),
+          [
+            const AddFolderWordsState.loaded([mockedWordEntityV1]),
```

Comment:
why did you change that?

Reply:
because now this state looks like : "  const factory AddFolderWordsState.loaded(List<WordsEntity> wordsList) = _Loaded;
"

Reply:
so why the const can't be outside of the list?